corehq/apps/case_importer/do_import.py

```diff
@@ -118,6 +119,10 @@ def do_import(self, spreadsheet):
     def import_row(self, row_num, raw_row):
         search_id = self._parse_search_id(raw_row)
         fields_to_update = self._populate_updated_fields(raw_row)
+        fields_to_update, error_messages = custom_case_import_operations(self.domain, row_num, raw_row,
+                                                                         fields_to_update)
+        if error_messages:
+            raise exceptions.CustomCaseRowError(message=", ".join(error_messages))
```

Comment:
Some error messages contain commas, if I am not mistaken. Would it make sense to use a semicolon as a delimiter here?
```suggestion
            raise exceptions.CustomCaseRowError(message="; ".join(error_messages))
```

Reply:
https://github.com/dimagi/commcare-hq/pull/30921/commits/e86cf2c56b1c67d2ab86f04580cca5ddfc7a1d76